folksonomy/api.py

```diff
@@ -868,3 +1102,104 @@ async def get_user_info(user: User = Depends(get_current_user)):
         "moderator": user_roles["moderator"],
         "user": user_roles["user"],
     }
+
+@app.post("/admin/value/rename", tags=["Admin - Value Management"])
+async def rename_value(
```

Comment:
I think we need a `check_value_clash` as we have a `check_property_clash` so that people can validate their operation before a rename.